packages/typescriptlang-org/lib/bootup/ingestion/createRootPagesLocalized.ts

```diff
@@ -44,6 +45,10 @@ export const createRootPagesLocalized = async (
       originalSitePath = originalSitePath.substring(0, originalSitePath.length - 3)
     }
 
+    // Always use /s for the path, because the join above in windows would be \
+    originalSitePath = originalSitePath.split("\\").join("/")
+    // fullpath = fullpath.split("\\").join("/");
```

Comment:
Todo: would you like me to send these as a separate PR for Windows support?

Reply:
Nah, if no-one's hit this issue (it only affects a few pages) it's fine

Reply:
Ok cool, filed https://github.com/microsoft/TypeScript-Website/issues/1043 for tracking. I'll remove these changes and try to remember to keep them just locally for my system.